ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/views/ManageSubscriptionsView.kt

```diff
@@ -187,52 +208,108 @@ private fun NoActiveSubscriptionsViewPreview() {
+    onButtonPress: (CustomerCenterConfigData.HelpPath) -> Unit,
     useOutlinedButton: Boolean = false,
 ) {
     Column {
         screen.supportedPaths.forEach { path ->
             ManageSubscriptionButton(
                 path = path,
-                onDetermineFlow = onDetermineFlow,
+                onButtonPress = onButtonPress,
                 useOutlinedButton = useOutlinedButton,
             )
         }
     }
 }
 
+@OptIn(ExperimentalPreviewRevenueCatPurchasesAPI::class)
+@Composable
+private fun OtherPlatformSubscriptionButtonsView(
+    localization: CustomerCenterConfigData.Localization,
+    support: CustomerCenterConfigData.Support,
+    managementURL: Uri?,
+    onAction: (CustomerCenterAction) -> Unit,
+) {
+    Column {
+        managementURL?.let {
+            CustomerCenterButton(
+                onClick = { onAction(CustomerCenterAction.OpenURL(it)) },
+                buttonContent = { modifier ->
+                    Text(
+                        text = localization.commonLocalizedString(
+                            CustomerCenterConfigData.Localization.CommonLocalizedString.MANAGE_SUBSCRIPTION,
+                        ),
+                        modifier = modifier,
+                        textAlign = TextAlign.Start,
+                        style = MaterialTheme.typography.bodyLarge,
+                    )
+                },
+            )
+        }
+        support.email?.let {
+            CustomerCenterButton(
+                onClick = { onAction(CustomerCenterAction.ContactSupport(it)) },
+                buttonContent = { modifier ->
+                    Text(
+                        text = localization.commonLocalizedString(
+                            CustomerCenterConfigData.Localization.CommonLocalizedString.CONTACT_SUPPORT,
+                        ),
+                        modifier = modifier,
+                        textAlign = TextAlign.Start,
+                        style = MaterialTheme.typography.bodyLarge,
+                    )
+                },
+            )
+        }
+    }
+}
+
 @OptIn(ExperimentalPreviewRevenueCatPurchasesAPI::class)
 @Composable
 private fun ManageSubscriptionButton(
     path: CustomerCenterConfigData.HelpPath,
-    onDetermineFlow: (CustomerCenterConfigData.HelpPath) -> Unit,
+    onButtonPress: (CustomerCenterConfigData.HelpPath) -> Unit,
     useOutlinedButton: Boolean,
 ) {
-    val buttonModifier = Modifier
+    CustomerCenterButton(
+        onClick = { onButtonPress(path) },
+        useOutlinedButton = useOutlinedButton,
+        buttonContent = { modifier ->
+            Text(
+                text = path.title,
+                modifier = modifier,
+                textAlign = TextAlign.Start,
+                style = MaterialTheme.typography.bodyLarge,
+            )
+        },
+    )
+}
+
+@OptIn(ExperimentalPreviewRevenueCatPurchasesAPI::class)
+@Composable
+private fun CustomerCenterButton(
```

Comment:
I wonder if this composable should be extracted to a separate file so it can be reused in different screens... Fine to do that in future PRs though.